src/api/index.js

```diff
@@ -25,6 +25,17 @@ export function api (command, args = {}) {
   })
 }
 
+export function apiPostForm (command, data = {}) {
```

Comment:
@RitchieVincent is there a way we can refactor the `api()` to do both get and post, for example refactor the `api` method to accept a `method` parameter that accepts GET by default, for example:
```
export function api (command, args = {}, method = 'GET') {
... snipped: refactor to send axios based request for any command/args/method ...
```

Reply:
Absolutely, I will get that sorted and update the PR.